.gitignore

```diff
@@ -77,3 +77,7 @@
 .ddev/homeadditions/
 .ddev/providers/
 .ddev/xhprof/
+
+# Ignore macOS DS_Store files
+.DS_Store
+**/.DS_Store
```

Comment:
Whilst we certinally should ignore these, should we instead encourgate devs to[ set up a global .gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)?

Reply:
Yes, I agree @andybroomfield that we should encourage global `.gitignore` usage; maybe we can add this somewhere in the getting started development documentation.

This change will also act as a failsafe should users not do the step.